SpringProject/src/main/java/com/gradproject2019/users/controller/UserController.java

```diff
@@ -19,6 +21,12 @@
     @Autowired
     private UserService userService;
 
+    @GetMapping(path = "/users")
```

Comment:
I think it's a good practice to provide an optional params for paginating this list (page, pageSize). If the number of users will increase dramatically, this endpoint will become unusable.

Reply:
At the moment the users will be listed in a scrollable dropdown box, and in the future it should be replaced by a search box. Would this mean that pagination is not required yet?

Reply:
In that case pagination is mandatory. Imagine that you have in your system 10K users, if so, all 10K users will be in this scroll box which will froze entire web page.

Reply:
Ok thanks i'll add in pagination, but still a bit confused about its implementation in the dropdown box - will the scrolling be replaced by pages of users?

Reply:
Yep, in the web you'll request batch of users to show in dropdown. First just 10-20 users, after that while scrolling another request for next 10 users and so on.

Reply:
Just a note that it might be complex (but necessary) to test the pagination on the endpoint, so I would recommend speaking to your BE tech lead on the scope of the tests.

I would expect there to be at least 1 test to test that you can navigate the pages (forwards and backwards) and that the page returns the user(s) you'd expect.